src/Microsoft.AspNetCore.Razor.Language/DefaultRazorIRLoweringPhase.cs

```diff
@@ -17,6 +17,9 @@ protected override void ExecuteCore(RazorCodeDocument codeDocument)
             var syntaxTree = codeDocument.GetSyntaxTree();
             ThrowForMissingDependency(syntaxTree);
 
+            var tagHelperContext = codeDocument.GetTagHelperContext();
```

Comment:
Agree with @ajaybhargavb here. I find that name overlap is incredibly confusing for us and for consumers.